### DISCUSSION TITLE
Access Neovim configuration from the dev shell?

### DISCUSSION THREAD

**Discussion by gueyenono** [nix-community/kickstart-nix.nvim#48] [Q&A] [answered]
First of all, thank you very much for such a great project, which is so helpful for noobs in the Nix ecosystem like me.

My question seems simple, but I'm not sure. What should I do to be able to access my neovim config from `nix develop`? Currently, neovim is not accessible from the dev shell.

Thank you

EDIT: I know that I can `nix build` then enter the dev shell with `nix develop .#default` and finally access Neovim with `./result/bin/nvim` but I would like to know if something can be done from the configuration to directly access Neovim with a simple `nvim` in the dev shell.

**Comment by kyp0717** [comment]
Thanks for asking.  I faced the same issue.   I'm new to nix (using for 2 months).  After entering the dev shell using `nix develop`, I can't access neovim.  Yet, in the flake, a default dev shell has been defined.   Am I missing something?

**Comment by mrcjkb** [comment, marked as answer]
Hey 👋 

You can add `pkgs.nvim-pkg` to the devShell's `buildInputs`.

**Reply by gueyenono** [reply]
> Hey 👋
> 
> You can add `pkgs.nvim-pkg` to the devShell's `buildInputs`.

Thank you for this solution. Actually, I had to repeat all the packages from the overlay in `./nix/neovim-overlay.nix` inside the shell in `./flake.nix`. I think this will eliminate the need to do so.
